### PR TITLE
feat(Resource-status/Listing): Add new column and allow to re-order them

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This uses the new Listing featres to allow the user to:
- Reorder the columns
- Add the following new columns: Monitoring server, fqdn, notification, check, alias

https://www.loom.com/share/62991338d3e4471789383f4b31b1319d?sharedAppSource=personal_library

**Fixes** # (issue)

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

**Comment by kduret** [comment]
Regarding video, dragging icon seems not vertical aligned with column label
By the way, this looks awesome

**Review by Thebarda — COMMENTED** [review]
It works very great 👌 
Is it possible to increase minimal width about information column (at least to display the column name) ?
https://www.loom.com/share/a5acf688ac59464aa9ba1a25a52efd4b

**Comment by bdauria** [comment]
> It works very great 👌
> Is it possible to increase minimal width about information column (at least to display the column name) ?
> https://www.loom.com/share/a5acf688ac59464aa9ba1a25a52efd4b

It’s actually difficult to to so, because the column widths are driven by the grid template of the Listing, which means that it can become eventually truncated when no sufficient space is remaining. 

We can work that out in another round though.

**Review by Thebarda — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16101,3 +16101,12 @@ msgstr "La date de fin doit être supérieure à la date de début"
...
 msgstr "Temps d'exécution"
+
+msgid "Notifications enabled"
+msgstr "Notifications activées"
+
+msgid "Notifications disabled"
+msgstr "Notifications désactivées"
+
+msgid "Checks disabled"
+msgstr "Vérifications désactivées"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16112]:
```suggestion
msgstr "Contrôles désactivées"
```

**Review by callapa — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -16101,3 +16101,12 @@ msgstr "La date de fin doit être supérieure à la date de début"
...
 msgstr "Temps d'exécution"
+
+msgid "Notifications enabled"
+msgstr "Notifications activées"
+
+msgid "Notifications disabled"
+msgstr "Notifications désactivées"
+
+msgid "Checks disabled"
+msgstr "Contrôles désactivées"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16112]:
```suggestion
msgstr "Contrôles désactivés"
```

**Review by kduret — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]
LGTM for the backend part

**Review by jdelpierre — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**